python/ray/serve/_private/router.py

```diff
@@ -358,6 +388,33 @@ def target_num_scheduling_tasks(self) -> int:
     def curr_replicas(self) -> Dict[str, ReplicaWrapper]:
         return self._replicas
 
+    def _get_actor_name_GCS_UNSAFE(self) -> str:
```

Comment:
@GeneDer @edoakes What do you think of this naming scheme (`_GCS_UNSAFE` suffix)? I wanted to make it very clear that this method isn't safe to call when the GCS is down.

Reply:
Personally I don't really like mixed casing in function names, would prefer it all lower cased if you really want to add it. Also feel just have that info in the comment should be sufficient which you already did. But not a hard blocker to me 🙂

Reply:
Yea info in the comment is fine -- there are many calls that can fail when the GCS is down and must be handled at the highest level

Reply:
Sounds good, I removed the suffix and left the comments as-is.